mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/gridfs/Photo.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.test.gridfs;
+
+import static org.hibernate.ogm.datastore.mongodb.options.BinaryStorageType.GRID_FS;
+import static org.hibernate.ogm.datastore.mongodb.test.gridfs.GridFsTest.BUCKET_NAME;
+
+import java.sql.Blob;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+
+import org.hibernate.ogm.datastore.mongodb.options.BinaryStorage;
+import org.hibernate.ogm.datastore.mongodb.options.GridFSBucket;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+
+@Entity
+public class Photo {
+	@Id
+	private String id;
+
+	private String description;
+
+	@Lob
+	@GridFSBucket(BUCKET_NAME)
+	@BinaryStorage( GRID_FS )
```

Comment:
Instead of using the annotation, as a first approach I would use the already existing `@Type` annotation something like:
```
@Lob
@Type(type = "gridfs")
private Blob content
```

Reply:
Why? Please see issue.

Reply:
I would prefer to use existing annotation instead of introducing new one.
Keep it like this for now, I will think about it

Reply:
Also, are there other types of BinaryStorage in MongoDB?

Reply:
No.  You can store binary content in document's field or in GridFS.